src/state/reducer.ts

```diff
@@ -13,18 +17,49 @@ import transactions from './transactions/reducer'
 import user from './user/reducer'
 import wallets from './wallets/reducer'
 
-export default {
-  application,
+const persistedReducers = {
   user,
   transactions,
   signatures,
+  lists,
+}
+
+const appReducer = combineReducers({
+  ...persistedReducers,
+  application,
   wallets,
   mint,
   mintV3,
   burn,
   burnV3,
   multicall: multicall.reducer,
-  lists,
   logs,
   [routingApi.reducerPath]: routingApi.reducer,
+})
+
+export type AppState = ReturnType<typeof appReducer>
+
+function createStorage() {
+  const db = localForage.createInstance({
+    name: 'redux',
+  })
+  return {
+    getItem: db.getItem,
+    setItem: db.setItem,
+    removeItem: db.removeItem,
+  }
```

Comment:
Why do you need to break this out?
```suggestion
  return db
```